src/main/resources/assets/securitycraft/lang/en_us.json

```
@@ -557,7 +557,7 @@
 	"help.securitycraft.universal_block_reinforcer_lvl3.info": "The Universal Block Reinforcer reinforces numerous vanilla blocks, and turns them into the SecurityCraft variants. To do this, right-click the Universal Block Reinforcer and put the vanilla block into the top slot, then close the GUI. To convert a reinforced block back to its vanilla variant, use the bottom slot. LvL3 has unlimited uses. Break any block outside of Creative mode to reinforce it.",
 	"help.securitycraft.briefcase.info": "The briefcase is a keypad-locked portable storage unit containing 12 slots. After crafting it, rightclick it to setup the 4-digit passcode to use from then on. Rightclick it again and insert the code to interact with the items. To remove a Briefcase's code or change its owner, put it into your offhand and rightclick while holding a Universal Key Changer/Universal Owner Changer in your mainhand.",
 	"help.securitycraft.universal_key_changer.info": "The universal key changer allows you to change the passcode of passcode-protected blocks you own. Rightclick a passcode-protected block to open a GUI, then type in the new passcode you want.",
-	"help.securitycraft.secret_sign_item.info": "The Secret Sign works like a vanilla sign, with the slight difference that only the owner of the sign and whitelisted players can view the text on it. Make sure noone can break the block it's placed on!",
+	"help.securitycraft.secret_sign_item.info": "The Secret Sign works like a vanilla sign, with the slight difference that only the owner of the sign and whitelisted players can view and interact with the text on it. Make sure noone can break the block it's placed on!",
```

```suggestion
	"help.securitycraft.secret_sign_item.info": "The Secret Sign works like a vanilla sign, with the slight difference that only the owner of the sign and whitelisted players can interact with, and view the text on it. Make sure noone can break the block it's placed on!",
```
Same thing as in the German file.